src/lib/providers/googleProvider.ts

```diff
@@ -0,0 +1,88 @@
+import got from 'got';
+import { AuthistOptions, User, UserInfo } from '../authist';
+import { createCredentials } from '../credentialsService';
+import { ERROR_CODE, NotAuthenticated } from '../error';
+import { handleError, sendRegistrationEmail } from '../providerUtils';
+
+export const PEOPLE_BASE_URL = 'https://people.googleapis.com';
+
+interface GoogleField {
```

Comment:
Is it possible to use some types for this from the People API?/Google SDK so we don't create a completely new type on our own?

Reply:
The problem is the `googleapis` does not have `@types`. We can add the whole `googleapis` lib but i didnt wont to do it because here is just one request. This is why I created this interface.

Reply:
sad :( this is good enough then :)